src/nagini_translation/extended_ast/translators/statement.py

```diff
@@ -0,0 +1,47 @@
+"""
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""
+
+import ast
+from typing import List
+
+from nagini_translation.lib.typedefs import Stmt
+from nagini_translation.translators.abstract import Context
+from nagini_translation.translators.statement import StatementTranslator
+from nagini_translation.lib.util import flatten, get_body_indices
+
+
+class ExtendedASTStatementTranslator(StatementTranslator):
+    """
+    Extended AST Version of the StatementTranslator
+    """
+    def _translate_return(self, node: ast.Return, ctx: Context) -> List[Stmt]:
+        rhs_stmt, rhs = self.translate_expr(node.value, ctx)
+        pos = self.to_position(node, ctx)
+        info = self.no_info(ctx)
+        if not ctx.result_var:
+            rvar = self.viper.NullLit(pos, info)
```

Comment:
Is this just a placeholder and going to be ignored or is it supposed to mean something? If it's just a placeholder, I'd just put None, if it's not, I don't think I understand what it means.